libpod/options.go

```diff
@@ -15,6 +16,7 @@ var (
 	ctrNotImplemented = func(c *Container) error {
 		return fmt.Errorf("NOT IMPLEMENTED")
 	}
+	nameRegex = regexp.MustCompile("[a-zA-Z0-9_-]+")
```

Comment:
NIT; Probably should have the [a-zA-Z0-9_-]+ as a constant.  Not sure if this could change in the future.